javascript/src/main/App.js

```diff
@@ -52,6 +53,8 @@ function App() {
           <AuthorizedRoute path="/member/analyzemessages/countmessages" exact component={CountMessagesByUser} authorizedRoles={["admin", "member"]} />
           <AuthorizedRoute path="/member/analyzemessages/messagehistogram" exact component={HistogramOfMessagesByUser} authorizedRoles={["admin", "member"]} />
           <AuthorizedRoute path="/member/analyzemessages/searchmessages" exact component={SearchMessagesByUser} authorizedRoles={["admin", "member"]} />
+          <AuthorizedRoute path="/member/analyzemessages/searchmessages" exact component={SearchMessagesByUser} authorizedRoles={["admin", "member"]} />
```

Comment:
is this copy pasted by accident?

Reply:
That was part of the original code, there were multiple duplicate lines. I may have deleted it and pushed it, but when we pulled from main it added that line back

Reply:
Ok, can you remove it, in main there is only one line, so maybe git got confused and thought you made a new line so it added a duplicate. https://github.com/ucsb-cs156-s21/proj-mapache-search/blob/main/javascript/src/main/App.js#L54